tests/tests_twobody/test_mean_elements.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from poliastro.twobody.mean_elements import get_mean_elements
+
+
+def test_get_mean_elements_raises_error_if_invalid_body():
+    body = "Sun"
```

Comment:
What about `body = "PlanetNine"`? Just kidding, looks good to me 👍🏽

Reply:
Yeah, better to keep the name away from the context of astronomical bodies :)